docs/operations/debug.md

```diff
@@ -38,6 +38,31 @@ spec:
 
 This option is best for debugging specific components.
 
+### libvirt virtqemud.conf set log_filters according to virt-launcher log Verbosity
+
+verbosity 5
+log_filters="3:remote 4:event 3:util.json 3:util.object 3:util.dbus 3:util.netlink 3:node_device 3:rpc 3:access 3:util.threadjob 3:cpu.cpu 3:qemu.qemu_monitor 3:qemu.qemu_monitor_json 3:conf.domain_addr 1:*"
+
+verbosity 6
+3:remote 4:event 3:util.json 3:util.object 3:util.dbus 3:util.netlink 3:node_device 3:rpc 3:access 3:util.threadjob 3:cpu.cpu 3:qemu.qemu_monitor 1:*
+
+verbosity 7
+3:remote 4:event 3:util.json 3:util.object 3:util.dbus 3:util.netlink 3:node_device 3:rpc 3:access 3:util.threadjob 3:cpu.cpu 1:*
+
+verbosity 8 and above
+3:remote 4:event 3:util.json 3:util.object 3:util.dbus 3:util.netlink 3:node_device 3:rpc 3:access 1:*
+
+### set libvirt log filter in VMI
```

Comment:
I believe this is related to fine tuning the defaults given above, so it shouldn't be a separate section.